apps/editing-toolkit/editing-toolkit-plugin/wpcom-tasks/index.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Atomic support for WPCOM onboarding checklist tasks.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE\WPCOMTasks;
+
+/**
+ * This logic is duplicated from Onboarding_Checklist\Blog
+ */
+function page_post_saved( $post_ID, $post, $update ) {
+	if ( get_option( 'show_on_front' ) === 'page' && (int) get_option( 'page_on_front' ) === $post_ID ) {
```

Comment:
We could also add a `is_atomic()` check here to make sure that this doesn't change the logic around this checklist item on simple sites

Reply:
I'd prefer to avoid the `is_initialized()` check if we can because it means we're depending on another option syncing across from wpcom to atomic before this code runs and even if it already works it makes me worry about a race condition. It also keeps this code simpler here.

You're right in that there's definitely a price for having this code be different from the action on wpcom but I think it's worth it in this instance.

Putting in inside the `is_atomic()` is an excellent point either way. Are you happy with that as a compromise?